proto/rill/runtime/v1/connectors.proto

```diff
@@ -197,3 +204,19 @@ message BigQueryListTablesResponse {
   string next_page_token = 1;
   repeated string names = 2;
 }
+
+message ScanConnectorsRequest {
+  string instance_id = 1;
+}
+
+message ScanConnectorsResponse {
+  repeated ScannedConnector connectors = 1;
+  map<string, string> existing_credentials = 2; // credentials already present in env file
```

Comment:
I don't think this belongs here. See other note about connector config and variables. We should either return *resolved* connector config. Or have an instance-level API to lookup all variables.